### PR TITLE
[WIP] 3-5 blog post and emberblog-template file

### PR DESCRIPTION
**Pull request by MelSumner**
(last edited by Jen)

### Blockers for publication:
- [x] release of the libraries themselves:
    - [x] ember.js
    - [x] ember-cli
    - [x] ember-data
- [x] finish draft (need to update ember data bc there are new features)
- [x] proofread by at least 1 person

### Content drafted:
- [x] ember.js
    - [x] https://github.com/emberjs/ember.js/blob/beta/CHANGELOG.md (bugfixes only, no new features) 
- [x] ember-cli
  - [x] Upgraded to Broccoli v2.0.0! 
  - [x] Migrated `./tmp` directory **out** of the project, into the default system temp directory.
  - [x] Migrated from `ember-cli-qunit` to `ember-qunit` as a dependency.
- [x] ember-data (will be a re-release of 3.4)
- [x] describe 3.4 LTS status

### CODE REVIEW

**Comment by rwjblue** [comment]
Added some bullet points RE: ember-cli@3.5

**Comment by jenweber** [comment]
cc @runspired here's the 3.5 release blog post

**Comment by jenweber** [comment]
I believe this blog post is more or less good to go, once these libraries are released.

If someone merges this, they should first check that all the links exist (they didn't when I added them) and update the post metadata to reflect the correct date.

File: source/blog/2018-10-08-ember-3-5-released.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+## Ember.js
+
+Ember.js is the core framework for building ambitious web applications.
+
+### Changes in Ember.js 3.5
+Ember.js 3.5 is an incremental, backwards compatible release of Ember with bugfixes. 
+It is common for minor releases to provide bugfixes that pave the way for
+release of new features in future releases.
+There are zero (0) new features, zero (0) deprecations, and five (5) minor
```
Comment (kategengler) [inline source/blog/2018-10-08-ember-3-5-released.md]: It's (2) bugfixes because the rest were all backported to 3.4.x

Comment (jenweber) [reply source/blog/2018-10-08-ember-3-5-released.md]: Edited.

File: source/blog/2018-10-08-ember-3-5-released.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+Today the Ember project is releasing version 3.5 of Ember.js, Ember Data, and Ember CLI. This release kicks off the 3.5 beta cycle for all sub-projects. We encourage our community (especially addon authors) to help test these beta builds and report any bugs before they are published as a final release in six weeks' time. The [ember-try](https://github.com/ember-cli/ember-try) addon is a great way to continuously test your projects against the latest Ember releases.
+
+Additionally, 3.4 is now promoted to LTS, which stands for Long
+Term Support. An LTS version of Ember continues to receive security updates
+for 9 release cycles (54 weeks)
+and bugfixes for 6 cycles (36 weeks).
+LTS releases typically occur every four minor versions, plus the
+first and last releases of major versions. 
+So far, the LTS releases for 3.x are 3.0 and 3.4.
```
[the remark this answers is not in the document]
Comment (kategengler) [reply source/blog/2018-10-08-ember-3-5-released.md]: Yes, the previous LTS before 3.4 is 2.18

[the remark this answers is not in the document]
Comment (kategengler) [reply source/blog/2018-10-08-ember-3-5-released.md]: The last release of a major version is also not necessarily an LTS

File: source/blog/2018-10-08-ember-3-5-released.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+## Ember.js
+
+Ember.js is the core framework for building ambitious web applications.
+
+### Changes in Ember.js 3.5
+Ember.js 3.5 is an incremental, backwards compatible release of Ember with bugfixes. 
+It is common for minor releases to provide bugfixes that pave the way for
+release of new features in future releases.
+There are zero (0) new features, zero (0) deprecations, and two (2) minor 
+bugfixes in this version. Three (3) bugfixes were backported to 3.4.
```
Comment (kategengler) [inline source/blog/2018-10-08-ember-3-5-released.md]: More bugfixes than that were backported to 3.4 -- might be better to just not mention? My earlier comment referred the number out of the 5 that were mentioned that were backported.

File: source/blog/2018-10-08-ember-3-5-released.md [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+Contributors to the Ember.js codebase itself should note that it now uses
+Typescript 3.0 internally.
+
+#### Bugfixes (2)
+
+To review the bugfixes made for 3.5, view the [Ember.js CHANGELOG](https://github.com/emberjs/ember.js/blob/master/CHANGELOG.md).
```
Comment (kategengler) [inline source/blog/2018-10-08-ember-3-5-released.md]: This links to the CHANGELOG on master, which will always be the latest CHANGELOG. 

Also, the CHANGELOG and the github release linked below will have identical information.

File: source/blog/2018-10-08-ember-3-5-released.md (hunk 3 above)
Comment (jenweber) [reply source/blog/2018-10-08-ember-3-5-released.md]: I will remove it. For my own learning, what does it mean for something to be backported? That it went in a 3.4 patch release?

**Comment by jenweber** [comment]
All corrections have been addressed. Thank you for the feedback!

I wanted to rearrange the order to put the most exciting project release first, but there were some concerns raised, so I’m leaving it. What I have done instead is added a summary sentence right at the beginning. This will help us in newsletter snippets and such, and improve the chances that people will keep reading. What I suggest is that we try it out, and if people don’t like it, drop the summary next time.

**Pull request merged**